### DISCUSSION TITLE
qiankun+Vue3+vite如何实现多页签缓存

### DISCUSSION THREAD

**Discussion by hongsusu** [umijs/qiankun#2608] [Q&A]
请问哪位大佬知道 qiankun+Vue3+vite如何实现多页签缓存，手动挂载的话页面是不是会卡顿，自定挂载应该怎么挂载

**Comment by jiangjunfeng98** [comment]
主应用缓存渲染容器，子应用缓存自己页面，可以参考 https://juejin.cn/post/7065559452908257311#heading-18
